pkg/provider/azure.go

```diff
@@ -310,6 +310,8 @@ type MultipleStandardLoadBalancerConfigurationSpec struct {
 // MultipleStandardLoadBalancerConfigurationStatus stores the properties regarding multiple standard load balancers.
 type MultipleStandardLoadBalancerConfigurationStatus struct {
 	ActiveServices sets.Set[string] `json:"activeServices" yaml:"activeServices"`
+
+	ActiveNodes sets.Set[string] `json:"activeNodes" yaml:"activeNodes"`
```

Comment:
Could you add a comment on what kinds of nodes are active?

Reply:
added